### PR TITLE
Renamed Create Profile to Install Wallet

### PR DESCRIPTION
**Pull request by stanislav-slavov**
ref #403

### CODE REVIEW

File: src/components/layout/Navigation.vue [hunk 1]
```diff
@@ -38,9 +38,9 @@
       class="navigation__item profile"
       @click="scrollTop()"
     >
-      <router-link :to="{ name: 'create-profile' }">
+      <router-link :to="{ name: 'install-wallet' }">
         <div class="navigation__item__image" />
-        <span>Create Profile</span>
+        <span>Install Wallet</span>
```
Comment (venimus) [inline src/components/layout/Navigation.vue:43]: May be it should be 'Install a wallet' - it is grammatically correct

Comment (venimus) [reply src/components/layout/Navigation.vue:43]: However the content is kind of a mess there. It says How to Use superhero
So probably we may just rename it Tutorial

**Comment by venimus** [comment]
It appears that the request is to completely remove this page. There will be a Tutorial elsewhere @mradkov did I got it right?

**Comment by mradkov** [comment]
> It appears that the request is to completely remove this page. There will be a Tutorial elsewhere @mradkov did I got it right?

Remove it from the navigation then. Keep the content we can move it in the FAQ in another PR.

**Pull request merged**